3party/expat/expat_impl.h

```diff
@@ -307,7 +307,7 @@ class CExpatImpl
   {
     assert (m_p != NULL);
     XML_SetUnknownEncodingHandler (m_p,
-      fEnable ? UnknownEncodingHandler : NULL);
```

Comment:
It looks like build fix, but not EnableUnknownEncodingHandler fix.
XML_SetUnknownEncodingHandler sets UnknownEncodingHandler and data pointer.
The first thing UnknownEncodingHandler does is data pointer dereference. So it's bad to pass NULL.
I think it's better to comment out EnableUnknownEncodingHandler than such fix.